osu.Game.Rulesets.Tau/TauRuleset.cs

```diff
@@ -110,6 +110,7 @@ public override IEnumerable<Mod> GetModsFor(ModType type)
             new KeyBinding(InputKey.MouseLeft, TauAction.LeftButton),
             new KeyBinding(InputKey.MouseRight, TauAction.RightButton),
             new KeyBinding(InputKey.Space, TauAction.HardButton),
+            new KeyBinding(InputKey.LShift, TauAction.HardButton),
```

Comment:
When simultaneously pressing both the bindings for hardbeats, only one pair of Pressed/Released events will be fired.

This is due to the `SimultaneousBindingMode` chosen [here](https://github.com/Altenhh/tau/blob/master/osu.Game.Rulesets.Tau/TauInputManager.cs#L28)

You'd need to change that if you want both the keybinds to share the same action. An alternative would be to give the second keybind its own action similar to the left and right buttons, but you'll also need to adjust `TauReplayFrame` to support the new action.